tests/settings/test_db.py

```diff
@@ -5,6 +5,7 @@ async def test_ensure(dbi, test_settings):
     settings = await virtool.settings.db.ensure(dbi)
 
     assert settings == {
```

Comment:
Removing the _id in Settings is tricky because in ensure and update I make a Settings object by unpacking the combined existing settings data and new settings data using Settings(**existing_and_new_info) so Settings needs to be able to accept an id parameter because it will be included in the existing settings document.

Reply:
Solutions:
* https://docs.python.org/3/library/stdtypes.html#dict.pop
* https://docs.mongodb.com/manual/tutorial/project-fields-from-query-results/#suppress-_id-field
   (use `False` for Python)